fedn/fedn/clients/reducer/restservice.py

```diff
@@ -181,22 +179,23 @@ def start():
                                        message='Warning. Reducer is not base-configured. please do so with config file.')
 
             if request.method == 'POST':
-
                 timeout = float(request.form.get('timeout'))
                 rounds = int(request.form.get('rounds', 1))
                 task = (request.form.get('task', ''))
                 clients_required = request.form.get('clients_required', 1)
                 clients_requested = request.form.get('clients_requested', 8)
 
                 #TODO: Enable in UI
-                validate = request.form.get('validate', True)
+                validate = request.form.get('validate', False)
```

Comment:
We do not want validate=False as the default.

Reply:
It's just for UI settings, It will not affect the default value, the default value is True, see return for real value